modules/core/shared/src/main/scala/lucuma/core/enums/F2ReadMode.scala

```diff
@@ -18,18 +19,18 @@ sealed abstract class F2ReadMode(
   val shortName: String,
   val longName: String,
   val description: String,
-  val minimumExposureTime: FiniteDuration,
-  val recommendedExposureTime: FiniteDuration,
-  val readoutTime: FiniteDuration,
+  val minimumExposureTime: TimeSpan,
+  val recommendedExposureTime: TimeSpan,
+  val readoutTime: TimeSpan,
```

Comment:
Replacing `FiniteDuration`, `Duration` and `NonNegDuration` where possible with `TimeSpan`.

Reply:
👍